### ISSUE TITLE
eslint cli is not working on relative path in 5.0.1 version

### ISSUE THREAD

**Issue by itsravindra** [eslint/eslint#10556]
**Tell us about your environment**

* **ESLint Version:** 5.0.1
* **Node Version:** 8.9.4
* **npm Version:** 6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```
{
  "env": {
    "browser": true
  },
  "globals": {
    "define": false,
    "require": false,
    "google": false,
    "Promise": false
  },
  "plugins": ["extra-rules", "no-storage", "requirejs"],
  "extends": "eslint:recommended",
  "rules": {
    "valid-jsdoc": 0,
    "require-jsdoc": 0,
    "no-undef-init": 2,
    "no-multi-spaces" : 2,
    "no-continue" : 2,
    "no-array-constructor" : 2,
    "no-undef": 2,
    "array-callback-return": 2,
    "complexity": ["error", { "max": 30 }],
    "no-unneeded-ternary": 2,
    "no-else-return" : 2,
    "no-useless-call": 2,
    "no-unused-expressions": 2,
    "no-unmodified-loop-condition": 2,
    "no-use-before-define": 2,
    "no-new-func": 2,
    "strict": 2,
    "no-multiple-empty-lines": 2,
    "no-mixed-operators": 2,
    "no-trailing-spaces": 2,
    "no-useless-concat": 2,
    "no-console": 2,
    "no-useless-escape": 0,
    "no-inline-comments": 2,
    "for-direction": 2,
    "no-alert": 2,
    "guard-for-in": 2,
    "no-empty-function": 2,
    "no-eval": 2,
    "no-multi-str": 2,
    "no-eq-null": 2,
    "eqeqeq": 2,
    "comma-dangle": 2,
    "semi": 2,
    "yoda": 2,
    "no-script-url": 2,
    "no-self-compare" : 2,
    "extra-rules/no-commented-out-code": 2,
    "quotes": ["error", "double"],
    "object-property-newline": 2,
    "dot-notation": 2,
    "no-storage/no-browser-storage": 2,
    "requirejs/no-js-extension": 2,
    "no-new-object" : 2,
    "no-lonely-if" : 2,
    "no-lone-blocks" : 2,
    "no-useless-return" : 2,
    "no-prototype-builtins" : 2,
    "getter-return" : 2,
    "no-implicit-globals" : 2,
    "no-self-compare" : 2,
    "no-shadow-restricted-names" : 2,
    "no-void" : 2
  }
}

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash
Not working
eslint ../components -c .eslintrc.json --fix --cache

Working
eslint components -c _build/.eslintrc.json --fix --cache
```

**What did you expect to happen?**
relative path in globing pattern can be the path of parent directory.

**What actually happened? Please include the actual, raw output from ESLint.**
D:\SVN\clip\core\channel\_build>eslint ../components -c .eslintrc.json --fix --cache

Oops! Something went wrong! :(

ESLint: 5.0.1.
No files matching the pattern "../components" were found.
Please check for typing mistakes in the pattern.

This same thing is working in 4.19

![image](https://user-images.githubusercontent.com/24821175/42272363-85f650bc-7fa3-11e8-8541-f10f16cceae6.png)
![image](https://user-images.githubusercontent.com/24821175/42272406-a02a8a0c-7fa3-11e8-83a1-a9475a43eafa.png)

**Comment by ilyavolodin** [comment]
Could you try this command line instead:
```
eslint "../components/**/*.js" -c .eslintrc.json --fix --cache
```

**Comment by craigklem** [comment]
Seeing the exact same issue in windows since upgrading to 5.x.

**Comment by itsravindra** [comment]
Fixed in 5.3.0
